src/main/java/com/tangerine/urlshortener/url/algorithm/Base62BaseAlgorithm.java

```diff
@@ -0,0 +1,40 @@
+package com.tangerine.urlshortener.url.algorithm;
+
+public class Base62BaseAlgorithm implements BaseAlgorithm {
+
+    private static final char[] tokens = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789".toCharArray();
+    private static final int TOKEN_LENGTH_BOUND = 62;
+    private static final int URL_LENGTH_BOUND = 8;
+    private static final String PADDING_TOKEN = "A";
+
+    @Override
+    public String encode(Long mappingId) {
+        StringBuilder sb = new StringBuilder();
+        while (mappingId > 0) {
+            sb.append(tokens[(int) (mappingId % TOKEN_LENGTH_BOUND)]);
+            mappingId /= TOKEN_LENGTH_BOUND;
+        }
+        return padding(sb);
+    }
+
+    @Override
+    public long decode(String shortUrl) {
+        long decodedId = 0L;
+        long factor = 1L;
+        for (int i = 0; i < URL_LENGTH_BOUND; i++) {
+            decodedId += String.valueOf(tokens).indexOf(shortUrl.charAt(i)) * factor;
+            factor *= TOKEN_LENGTH_BOUND;
+        }
+        return decodedId;
+    }
+
+    private String padding(StringBuilder sb) {
+        int length = sb.length();
+        while (length < URL_LENGTH_BOUND) {
+            sb.append(PADDING_TOKEN);
+            length++;
+        }
+        return sb.toString();
+    }
+
```

Comment:
모든 코드가 따박따박 붙어있는것보다 문맥에 따라 개행을 해주면 가독성을 챙길 수 있어요 

```suggestion

    @Override
    public String encode(Long mappingId) {
        StringBuilder sb = new StringBuilder();
        
        while (mappingId > 0) {
            sb.append(tokens[(int) (mappingId % TOKEN_LENGTH_BOUND)]);
            mappingId /= TOKEN_LENGTH_BOUND;
        }
        
        return padding(sb);
    }

    @Override
    public long decode(String shortUrl) {
        long decodedId = 0L;
        long factor = 1L;
        
        for (char c : shortUrl.toCharArray()) {
          decodedId += String.valueOf(tokens).indexOf(c) * factor;
          factor *= TOKEN_LENGTH_BOUND;
       }
        
        return decodedId;
    }

    private String padding(StringBuilder sb) {
        int length = sb.length();
        
        while (length < URL_LENGTH_BOUND) {
            sb.append(PADDING_TOKEN);
            length++;
        }
        
        return sb.toString();
    }

```